sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient_helpers.cs

```diff
@@ -354,5 +375,63 @@ private string GenerateTokenFromAzureKeyCredential(DateTimeOffset expiresAt, str
 
             return jwt.BuildString();
         }
+
+        private string GetRelativeClientEndpoint(ClientEndpointType endpointType) => endpointType switch
+        {
+            ClientEndpointType.Default => $"client/hubs/{_hub}",
+            ClientEndpointType.MQTT => $"clients/mqtt/hubs/{_hub}",
+            _ => throw new ArgumentOutOfRangeException(nameof(endpointType))
+        };
+
+        private async Task<string> GetClientAccessTokenCore(
+             DateTimeOffset expiresAt,
+             string userId,
+             IEnumerable<string> roles,
+             IEnumerable<string> groups,
+             ClientEndpointType clientEndpointType,
+             bool async,
+             CancellationToken cancellationToken = default)
+        {
+            if (clientEndpointType == ClientEndpointType.MQTT && _apiVersionEnum < WebPubSubServiceClientOptions.ServiceVersion.V2024_01_01)
```

Comment:
maybe simply only keep this version V2024_01_01

Reply:
Do you mean removing the old version? The `ServiceVersion` enum class is public and deleting a value is breaking change.